lib/Admin.php

```diff
@@ -607,10 +607,17 @@ public function add_image_to_library(): void {
 	public static function get_widen_url_path( $url ) {
 		$url_arr  = wp_parse_url( $url );
 		$url_path = $url_arr['path'];
+		$path     = '';
 
-		$path = ltrim( $url_path, '/wp-content/uploads/' );
```

Comment:
ltrim was giving us unexpected results for certain URL's. Replaced with preg_replace() and also added a missing check for multisite.